### PR TITLE
[Issue #544] Hide/show Time series histogram with a new button

### PR DESCRIPTION
**Pull request by dongchime**
I introduced a new button on the map interface to hide/show the time series histogram.

_**examples/twittermap/web/app/views/twittermap/index.scala.html**_
1. Added a button div
2. Added a javascript code to hide/show the time series histogram and the number of tweets.

_**examples/twittermap/web/app/views/twittermap/main.scala.html**_
1. Added the jquery-ui library as a dependency.

**_examples/twittermap/web/public/stylesheets/main.css_**
1. Changed the appearance of the new button with css. (.slide-up-down)

Implementation: https://youtu.be/vwaKrJHOKt8

**Issue: #544**

### CODE REVIEW

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 1]
```diff
@@ -25,6 +25,15 @@
    <search-bar></search-bar>
    <predefined-keywords></predefined-keywords>
 
+  <!-- modifiy hide and show button -->
+
+  <!-- <button type="button" class="slide-up-down">down</button> -->
```
Comment (baiqiushi) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: Let's remove useless comments. Only keep comment for current implementation of this button.

File: examples/twittermap/project/dependencies.scala [hunk 2]
```diff
@@ -67,7 +67,8 @@ object Dependencies {
     "org.webjars" % "font-awesome" % "4.5.0",
     "org.webjars.bower" % "bootstrap-vertical-tabs" % "1.2.1",
     // other module
-    "com.fasterxml.jackson.core" % "jackson-core" % "2.9.4"
+    "com.fasterxml.jackson.core" % "jackson-core" % "2.9.4",
+    "org.webjars.bower" % "jquery-ui" % "1.12.1"
```
Comment (waans11) [inline examples/twittermap/project/dependencies.scala]: Put a comment above the line 71
// Added jquery-ui for showing/hiding the time series histogram.

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 3]
```diff
@@ -25,6 +25,11 @@
    <search-bar></search-bar>
    <predefined-keywords></predefined-keywords>
 
+  <div>
```
Comment (waans11) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: Also, put a comment above about this button. Describe what this button does.

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 4]
```diff
@@ -78,9 +83,24 @@ <h1> About </h1>
...
 
-
-
 </div>
 
-
 }
+
+<script>
+
```
Comment (waans11) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: No space (blank line) is required.

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 5]
```diff
@@ -78,9 +83,24 @@ <h1> About </h1>
...
+        $( 'button span' ).removeClass( 'glyphicon-menu-down' ).addClass('glyphicon-menu-up');
+        $(this).animate({bottom: '5px'});
+      }
+      else {
+        $( 'button span' ).removeClass( 'glyphicon-menu-up').addClass( 'glyphicon-menu-down');
+        $(this).animate({bottom: '110px'}); 
+      }
+    });
+  });
+
```
Comment (waans11) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: No space (blank line) is required.

File: examples/twittermap/web/app/views/twittermap/index.scala.html [hunk 6]
```diff
@@ -78,9 +83,24 @@ <h1> About </h1>
...
-
-
 </div>
 
-
 }
+
+<script>
+
+  $(document).ready(function() {
```
Comment (waans11) [inline examples/twittermap/web/app/views/twittermap/index.scala.html]: Put a comment about this function before line 92. What does this function do? And how does it work?

File: examples/twittermap/web/public/stylesheets/main.css [hunk 7]
```diff
@@ -222,4 +222,16 @@ time-series {
   color: #3b94d9; }
 
 .tweet-link:hover {
-  text-decoration: underline; }
\ No newline at end of file
+  text-decoration: underline; }
+
+.slide-up-down {
```
Comment (waans11) [inline examples/twittermap/web/public/stylesheets/main.css]: Please also put a CSS style comment /* */ here about this. What is this about.

**Comment by chenlica** [comment]
@dongchime Please embed the Youtube as a "big image" (instead of just a URL) to make it more visible and easy to click.  Thanks.

**Pull request merged**